templates/locales/en.yaml

```diff
@@ -1,12 +1,12 @@
-# Footer 
+# Footer
 # (for all emails)
 FooterOpenSource: "An open source, not-for-profit effort to build an open data platform and better applications that reduce the burden of diabetes."
 FooterGetSupport: "Get Support"
 FooterSincerely: "Sincerely,"
 FooterSignature: "The Tidepool Team"
 FooterMadePossible: "Made possible by"
```

Comment:
```suggestion

```

Reply:
not needed in final version but not sure what will happen to these lines when merged with master

Reply:
this should be dropped from fr.yaml as well?